applications/LinearSolversApplication/README.md

```diff
@@ -168,36 +168,38 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl) (see belo
     **Linux:**
 
     Set the environment before using MKL
+    
     ```bash
-    source /opt/intel/mkl/bin/mklvars.sh intel64 lp64
+    source /opt/intel/oneapi/setvars.sh intel64
     ```
 
 ## Install MKL on Ubuntu with apt
 
-Intel MKL can be installed with apt on Ubuntu. A guide can be found in https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo.
-For example to install the MKL 2020 version
+Intel MKL can be installed with apt on Ubuntu. A guide can be found in [here](https://neelravi.com/post/intel-oneapi-install/).
+For example to install the MKL 2022 version
 
 ```bash
 sudo bash
 # <type your user password when prompted.  this will put you in a root shell>
-# cd to /tmp where this shell has write permission
-cd /tmp
-# now get the key:
-wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-# now install that key
-apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-# now remove the public key file exit the root shell
-rm GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-# add the repository for mkl only (other packages are not needed here)
-sh -c 'echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list'
-apt-get update -y
-# install specific version of intel-mkl
-apt-get install -y intel-mkl-2020.4-912
+# If they are not installed, you can install using the following command:
+sudo apt update
+sudo apt -y install cmake pkg-config build-essential
+# use wget to fetch the Intel repository public key
+wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# add to your apt sources keyring so that archives signed with this key will be trusted.
+sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# remove the public key
+rm GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# Configure apt client to use Intel repository
+sudo add-apt-repository "deb https://apt.repos.intel.com/oneapi all main"
+# Install all HPC related dependencies
+sudo apt install intel-hpckit
```

Comment:
this is much larger than mkl, like several GB!

Can you please add a separate section instead?

Reply:
I can only put the MKL one...

Reply:
Happy?